doc/api/async_hooks.md

```diff
@@ -460,6 +460,59 @@ init for PROMISE with id 6, trigger id: 5  # the Promise returned by then()
   after 6
 ```
 
+#### async_hooks.executionAsyncResource()
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object} The resource that triggered the current
+  execution context.
+  Useful to store data within the resource.
+
+```js
+const { open } = require('fs');
+const { executionAsyncId, executionAsyncResource } = require('async_hooks');
+
+console.log(executionAsyncId(), executionAsyncResource());  // 1 null
```

Comment:
is `1 null` still true as an empty object is set in `Environment` constructor?

Reply:
Ah, nope. Good catch. 🙂

Reply:
Fixed.